### ISSUE TITLE
Example from README does not work

### ISSUE THREAD

**Issue by priya2811-khatri** [dsaltares/fetch-gh-release-asset#5]
I tried to fetch an assets from another repo by using the Readme Example.
```
uses: dsaltares/fetch-gh-release-asset@master
with:
  repo: “Organisation/Repo”
  version: "latest"
  file: "asset_01.zip"
  token: ${{ secrets.MY_TOKEN }}
```
Job failed with an error of `Missing GITHUB_TOKEN env variable`.

Then I tried with following method:
```
uses: dsaltares/fetch-gh-release-asset@master
       env:
        GITHUB_TOKEN: ${{ secrets.MY_TOKEN }}
       with:
        repo: "Organisation/Repo"
        version: "latest"
        file: "asset_01.zip"
        token: ${{ env.GITHUB_TOKEN }}
```
And it worked.

**Comment by dsaltares** [comment]
The `GITHUB_TOKEN` env variable is no longer required as of https://github.com/dsaltares/fetch-gh-release-asset/pull/8. However, one does need to specify it via command line or via action options. I will update the README.md example.

**Comment by dsaltares** [comment]
Fixed by #1b96af4179c7d64fde543758dff8ca6f6df4cd77.

**Comment by dsaltares** [comment]
Thanks for reporting!
